Templates/FaceLocator-CSharp/run.csx

```diff
@@ -0,0 +1,90 @@
+﻿// Setup
+// 1) Go to https://www.microsoft.com/cognitive-services/en-us/computer-vision-api 
+//    Sign up for computer vision api
+// 2) Go to Function app settings -> App Service settings -> Settings -> Application settings
+//    create a new app setting FACEAPI_Subscription_Key and use Computer vision key as value
+#r "Microsoft.WindowsAzure.Storage"
+#r "Newtonsoft.Json"
+
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using Newtonsoft.Json;
+using Microsoft.WindowsAzure.Storage.Table;
+using System.IO;
+
+public static async Task Run(Stream image, string name, ICollector<FaceRectangle> outTable, TraceWriter log)
+{
+    string result = await CallFaceAPI(image);
+    log.Info(result);
+
+    if (String.IsNullOrEmpty(result))
+    {
+        return;
+    }
+
+    ImageData imageData = JsonConvert.DeserializeObject<ImageData>(result);
+    foreach (Face face in imageData.faces)
+    {
+        FaceRectangle faceRectangle = face.faceRectangle;
+        faceRectangle.RowKey = Guid.NewGuid().ToString();
+        faceRectangle.PartitionKey = "Functions";
+        faceRectangle.imageFile = name + ".jpg";
+        outTable.Add(faceRectangle);
+    }
+}
+
+static async Task<string> CallFaceAPI(Stream image)
+{
+    using (var client = new HttpClient())
+    {
+        var content = new StreamContent(image);
+        var url = "https://api.projectoxford.ai/vision/v1.0/analyze?visualFeatures=Faces";
+        client.DefaultRequestHeaders.Add("Ocp-Apim-Subscription-Key", Environment.GetEnvironmentVariable("FACEAPI_Subscription_Key"));
+        content.Headers.ContentType = new MediaTypeHeaderValue("application/octet-stream");
+        var httpResponse = await client.PostAsync(url, content);
+
+        if (httpResponse.StatusCode == HttpStatusCode.OK)
+        {
+            return await httpResponse.Content.ReadAsStringAsync();
+        }
+    }
+    return null;
+}
+
+public class ImageData
+{
+    [JsonProperty("faces")]
+    public List<Face> faces { get; set; }
+}
+
+public class Face
+{
+
+    [JsonProperty("age")]
+    public int age { get; set; }
```

Comment:
Removing the JSON attributes would clear up a lot of the code in the sample, though. As the types are defined today, they're not really required (except for maybe a JsonIgnore)